ceph_deploy/util/pkg_managers.py

```diff
@@ -319,6 +330,16 @@ class DNF(RPMManagerBase):
     executable = 'dnf'
     name = 'dnf'
 
+    def install(self, packages, **kw):
+        extra_install_flags = kw.pop('extra_install_flags', [])
+        if '--best' not in extra_install_flags:
```

Comment:
will there never be a situation where we do not want `--best` ? (because it looks like we are forcing it regardless)